browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -10493,24 +10458,10 @@ index 026121d114c5fcad84c1396be8d692625beaa3bd..edd6e5cae033124c589959a42522fde0
  }
  #endif
 diff --git a/Source/WebKit/UIProcess/API/C/WKPage.cpp b/Source/WebKit/UIProcess/API/C/WKPage.cpp
-index 20a1e302ddbd2a61e3c5a2cd5ef11d35c94bffba..f682c0b9d17c47a6b9dd410e1f42c2bf0d96f6ae 100644
+index 6047c5fa4e736867566222d63f66572af94cd810..2028199bd8a10b9fb27d89acb25992c852725b2f 100644
 --- a/Source/WebKit/UIProcess/API/C/WKPage.cpp
 +++ b/Source/WebKit/UIProcess/API/C/WKPage.cpp
-@@ -1775,6 +1775,13 @@ void WKPageSetPageUIClient(WKPageRef pageRef, const WKPageUIClientBase* wkClient
-             completionHandler(String());
-         }
- 
-+        void handleJavaScriptDialog(WebPageProxy& page, bool accept, const String& value) final {
```

Comment:
Removing this will break our dialog handling on windows, I commented in the pretty patch on this.